src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -1,12 +1,31 @@
 package wolox.training.controllers;
 
+import java.util.List;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
+import wolox.training.exceptions.responses.BookIdMismatchException;
+import wolox.training.exceptions.responses.BookNotFoundException;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
 
 @Controller
+@RequestMapping("/api/training/books")
 public class BookController {
+    private static final String MSG = "Book not found";
```

Comment:
I recommend trying to use more descriptive names, for example you could name this one `BOOK_NOT_FOUND_MSG`, so if later on you need to add more, their names will be clearer 😄

Reply:
thanks, in some of the next PR i made that change but i did not put "_" to separate, i going to made the suggest, thank you :smile: